rasa/nlu/extractors/extractor.py

```diff
@@ -164,13 +166,28 @@ def convert_predictions_into_entities(
                 )
                 entities.append(entity)
             else:
-                entities[-1][ENTITY_ATTRIBUTE_END] = token.end
-                if confidences is not None:
-                    self._update_confidence_values(entities, confidences, idx)
+                if last_token_end == token.start - 1:
+                    entities[-1][ENTITY_ATTRIBUTE_END] = token.end
+                    if confidences is not None:
+                        self._update_confidence_values(entities, confidences, idx)
+                else:
```

Comment:
can this `if/else` be indented to the left, so that we have
```python
if new_tag_found:
    ...
elif last_token_end == token.start - 1:
    ...
else:
    ...
```